src/main/java/li/cil/tis3d/common/init/BootstrapCommon.java

```diff
@@ -18,25 +20,30 @@
 import net.fabricmc.loader.api.FabricLoader;
 
 import java.io.File;
+import java.util.List;
 
 @SuppressWarnings("unused")
 public final class BootstrapCommon implements ModInitializer {
     @Override
     public void onInitialize() {
         // Load our settings first to have all we need for remaining init.
-        Settings.load(new File(FabricLoader.getInstance().getConfigDirectory(), API.MOD_ID + ".cfg"));
+        Settings.load(new File(FabricLoader.getInstance().getConfigDirectory(), CommonAPI.MOD_ID + ".cfg"));
 
         // Initialize API.
         //noinspection Convert2MethodRef
-        API.itemGroup = FabricItemGroupBuilder.create(Constants.NAME_ITEM_GROUP).
+        LocalAPI.common.itemGroup = FabricItemGroupBuilder.create(Constants.NAME_ITEM_GROUP).
```

Comment:
I'm accessing a `common` here that is still null, sorry about that; last minute untested code changes bite me once again.

Reply:
Ohh, I did not catch that. Sneaky.